scripts/project_stats.ipynb

```diff
@@ -0,0 +1,174 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Compute Project Stats\n",
+    "\n",
+    "* This notebook uses the GitHub [GraphQL API](https://developer.github.com/v4/) to compute the number of open and \n",
+    "  closed bugs pertaining to Kubeflow GitHub Projects\n",
+    "  * Stats are broken down by labels\n",
+    "* Results are plotted using [plotly](https://plot.ly)\n",
+    "  * Plots are currently published on plot.ly for sharing; they are publicly vieable by anyone\n",
+    "  \n",
+    "## Setup GitHub\n",
+    "\n",
+    "* You will need a GitHub personal access token in order to use the GitHub API\n",
+    "* See these [instructions](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) for creating a personal access token\n",
+    "  * You will need the scopes:\n",
+    "    * repo\n",
+    "    * read:org    \n",
+    "* Set the environment variable `GITHUB_TOKEN` to pass your token to the code\n",
+    "\n",
+    "## Setup Plot.ly Online\n",
+    "\n",
+    "* In order to use plot.ly to publish the plot you need to create a plot.ly account and get an API key\n",
+    "* Follow plot.ly's [getting started guide](https://plot.ly/python/getting-started/)\n",
+    "* Store your API key in `~/.plotly/.credentials `"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 177,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use plotly cufflinks to plot data frames\n",
+    "# https://plot.ly/ipython-notebooks/cufflinks/\n",
+    "# instructions for offline plotting\n",
+    "# https://plot.ly/python/getting-started/#initialization-for-offline-plotting\n",
+    "#\n",
+    "# Follow the instructions for online plotting:\n",
+    "# https://plot.ly/python/getting-started/\n",
+    "# You will need to setup an account\n",
+    "import plotly\n",
+    "import plotly.plotly as py\n",
+    "import plotly.graph_objs as go\n",
+    "import cufflinks as cf\n",
+    "from importlib import reload\n",
+    "import itertools"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 194,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<module 'project_stats' from '/home/jlewi/git_kubeflow-community/scripts/project_stats.py'>"
```

Comment:
is this intended to run for everyone? If so, might need to replace /home/jlewi/

Reply:
This is just the output value. So when run on a different machine it would output a different value.